tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py36
 deps =
   mypy
   pytest
+  pip~=20.0
```

Comment:
This probably should be `>=`, no?

Reply:
I usually like to lock my package requirements to a major release with `~=`. If you think we'd be ok with any future pip release I can change it to `>=`, as suggested.

Reply:
Why do you add the pip-dependency at all? CI works fine as it is and pip ist installed on default anyway

Reply:
What changed is that PyQt started to use a [manylinux2014](https://www.python.org/dev/peps/pep-0599/) wheel tag (the previous manylinux1 tag was always inaccurate). However, support for that was only added in [pip 19.3](https://github.com/pypa/pip/blob/master/NEWS.rst#193-2019-10-14).